### PR TITLE
Redirect to `next` on login success.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: mozilla_django_oidc/views.py [hunk 1]
```diff
@@ -70,6 +71,12 @@ def __init__(self, *args, **kwargs):
     def get(self, request):
         """OIDC client authentication initialization HTTP endpoint"""
         state = get_random_string(import_from_settings('OIDC_STATE_SIZE', 32))
+        redirect_field_name = import_from_settings('OIDC_REDIRECT_FIELD_NAME', 'next')
```
Comment (akatsoulas) [inline mozilla_django_oidc/views.py]: Please add OIDC_REDIRECT_FIELD_NAME in env-dist

File: mozilla_django_oidc/views.py [hunk 2]
```diff
@@ -70,6 +71,12 @@ def __init__(self, *args, **kwargs):
...
+
+        if redirect_field_name in request.GET:
+            request.session['oidc_login_next'] = request.GET[redirect_field_name]
+        else:
+            request.session['oidc_login_next'] = None
```
Comment (akatsoulas) [inline mozilla_django_oidc/views.py]: You could replace this block with request.GET.get('redirect_field_name')

File: tests/test_views.py [hunk 3]
```diff
@@ -49,6 +49,36 @@ def test_get_auth_success(self):
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, '/success')
 
+    @override_settings(LOGIN_REDIRECT_URL='/success')
+    def test_get_auth_success_next_url(self):
+        """Test successful callback request to RP with custom `next` url."""
+        user = User.objects.create_user('example_username')
+        user.is_active = True
```
Comment (akatsoulas) [inline tests/test_views.py]: You can pass directly keyword args in `create_user`.

File: tests/test_views.py [hunk 4]
```diff
@@ -49,6 +49,36 @@ def test_get_auth_success(self):
...
+        with patch('mozilla_django_oidc.views.auth.authenticate') as mock_auth:
+            with patch('mozilla_django_oidc.views.auth.login') as mock_login:
+                mock_auth.return_value = user
+                response = callback_view(request)
+
+                mock_auth.assert_called_once_with(code='example_code', state='example_state')
+                mock_login.assert_called_once_with(request, user)
+
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(response.url, '/foobar')
```
Comment (akatsoulas) [inline tests/test_views.py]: On second thought, you can omit this test altogether and add only the last two assert in `test_get_auth_success` test.

Comment (johngian) [inline tests/test_views.py]: Here we are testing the functionality that overrides the default success URL.

**Comment by johngian** [comment]
Fixes #51

**Pull request merged**